free-2.md

```diff
@@ -58,12 +169,38 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this might actually be a fairly feasible project. It looks like Paul
+Dapolito took a crack at it last year, but only targeted problem sets and
+memos. I think it would be good to take a more general approach, and that this
+would actually make for a very good DSLs project.
+
+I imagine most of the time would be spent learning a lot about the way LaTeX
+works and then deciding what parts of it are acceptable to cut out for the sake
+of nicer syntax and more readable code (I can see verbatim being problematic).
```

Comment:
Ah, indeed. I agree understanding the rendering process is probably quite the project. I think I could settle for understanding the parser/macro expansion system.